gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/ApiKeyEntity.java

```diff
@@ -29,13 +33,9 @@
 
     private String key;
 
-    private String subscription;
-
-    private String application;
+    private List<SubscriptionEntity> subscriptions;
```

Comment:
Maybe we could initialize all those subscriptions lists to empty lists, in declaration or in builder ?
That would avoid a lot of potential NPE.

```suggestion
    private List<SubscriptionEntity> subscriptions = new ArrayList();
```